tinygrad/state.py

```diff
@@ -45,6 +45,17 @@ def get_state_dict(obj, prefix:str='', tensor_type=Tensor) -> Dict[str, Tensor]:
 def get_parameters(obj) -> List[Tensor]: return list(get_state_dict(obj).values())
 
 def load_state_dict(model, state_dict, strict=True):
+  if 'model.embed_tokens.weight' in state_dict: # HuggingFace transformers format
+    hf_keymap = {
```

Comment:
This absolutely doesn't belong here

Reply:
Apologies, had forgotten this wasn't llama-specific, moved to Llama.build

Reply:
Reverted this file, it's now all in llama.py